src/clp_ffi_py/ir/native/PySerializer.cpp

```
@@ -0,0 +1,560 @@
+
+    return PyLong_FromSsize_t(num_byte_serialized.value());
+}
+
+CLP_FFI_PY_METHOD auto PySerializer_get_num_bytes_serialized(PySerializer* self) -> PyObject* {
+    return PyLong_FromSsize_t(self->get_num_bytes_serialized());
+}
+
+CLP_FFI_PY_METHOD auto PySerializer_flush(PySerializer* self) -> PyObject* {
+    if (false == self->flush()) {
+        return nullptr;
+    }
```

cppcheck

[error] Found an exit path from function with non-void return type that has missing return statement

(missingReturn)